zio-http/src/main/scala/zhttp/service/RequestBodyHandler.scala

```diff
@@ -0,0 +1,21 @@
+package zhttp.service
+import io.netty.channel.ChannelHandler.Sharable
+import io.netty.channel.{ChannelHandlerContext, SimpleChannelInboundHandler}
+import io.netty.handler.codec.http.HttpContent
+import zhttp.http.HttpData.{UnsafeChannel, UnsafeContent}
+
+@Sharable
+final class RequestBodyHandler[R](
+  var callback: (UnsafeChannel, Int) => UnsafeContent => Unit,
```

Comment:
```suggestion
  var callback: (UnsafeChannel) => UnsafeContent => Unit,
```

The size/limits should be managed by the user of this API.